tests/unit/index.test.js

```diff
@@ -197,6 +197,43 @@ Tape('start with true filter with write to file with non-JSON', (t) => {
   })
 })
 
+Tape('start with true filter with write to file with non-JSON with isJson=false', (t) => {
+  beforeEach()
+  t.plan(4)
+
+  const writeStub = Sinon.stub()
+  const endStub = Sinon.stub()
+  const rfs = {
+    createStream: () => ({
+      write: writeStub,
+      end: endStub
+    })
+  }
+
+  Mock('rotating-file-stream', rfs)
+
+  const stdin = StdInMock.stdin()
+  StdOutMock.use({ stdout: true, stderr: true, print: true })
+
+  const config = { filter () { return true }, output: { isJson: false, isRawOutput: true, path: 'test.log', options: {} }, exit: false }
```

Comment:
Please remove this option:
```suggestion
  const config = { filter () { return true }, output: { isJson: false, path: 'test.log', options: {} }, exit: false }
```

Reply:
ah, good catch. missed it when i created this change set off of the last

Reply:
updated